ondemand/action.go

```diff
@@ -0,0 +1,255 @@
+package ondemand
+
+import (
+	"context"
+	
+	"encoding/json"
+	"os"
+	"errors"
+
+	"strconv"
+	
+	"github.com/project-flogo/flow"
+	"github.com/project-flogo/flow/definition"
+	"github.com/project-flogo/flow/instance"
+	"github.com/project-flogo/core/data"
+	"github.com/project-flogo/flow/model"
+	"github.com/project-flogo/core/data/expression"
+	_ "github.com/project-flogo/flow/model/simple"
+	flowsupport "github.com/project-flogo/flow/support"
+	"github.com/project-flogo/core/action"
+	"github.com/project-flogo/core/data/resolve"
+	"github.com/project-flogo/core/data/metadata"
+	"github.com/project-flogo/core/data/mapper"
+	"github.com/project-flogo/core/support/log"
+	"github.com/project-flogo/core/support"
+)
+
+const (
+	FLOW_REF = "github.com/project-flogo/contrib/action/flow/ondemand"
```

Comment:
This should probably go away or be updated to the new location github.com/project-flogo/flow/ondemand